applications/physics/cosmology/cosmoflow/ref/README.md

```diff
@@ -0,0 +1,3 @@
+# PyTorch Reference Implementation of CosmoFlow
+
+This directory contains a reference implementation of CosmoFlow in PyTorch as it is implemented for LBANN in the parent directory. This implementation uses `DistributedDataParallel` (DDP) to run on multiple GPUs. The included `batch.sh` script provides an example for training this model with DDP on Lassen.
```

Comment:
You might also note that the script hardcodes OpenMPI env variables, so even if you write a new batch script, it still may not run on other systems.

Reply:
Fixed in https://github.com/LLNL/lbann/pull/2199/commits/05265916be3ce2adafa5bf68da350d6e3d680b14